archivist/compliance.py

```diff
@@ -26,7 +26,7 @@
 
 # pylint:disable=unused-import      # To prevent cyclical import errors forward referencing is used
 # pylint:disable=cyclic-import      # but pylint doesn't understand this feature
-import archivist as type_helper
+from . import archivist as type_helper
```

Comment:
this is correct - other files use a different import statement...

Reply:
Changed all imports to match this